src/mixins/itext_behavior.mixin.js

```diff
@@ -562,26 +560,21 @@
       var canDrop = !e.defaultPrevented && this.canDrop(e);
       if (!this.__isDraggingOver && canDrop) {
         this.__isDraggingOver = true;
-        this.enterEditing(e);
-        this.__isDragging && this.abortCursorAnimation();
       }
       else if (this.__isDraggingOver && !canDrop) {
         //  drop state has changed
         this.__isDraggingOver = false;
         !this.__isDragging && this.clearContextTop();
-        this.exitEditing();
       }
-      if (this.__isDraggingOver) {
+      if (this.__isDraggingOver && canDrop) {
```

Comment:
are you sure about this?

Reply:
i think this is a consequence of the previous ifs, and i made explicit.
I didn't think of it too much, could be wrong.

Reply:
oh i see is not the same because this just an if and not an else if.
needs to be removed